lua/entities/gmod_wire_pod.lua

```diff
@@ -482,7 +482,7 @@ function ENT:Think()
 		-- Button pressing
 		if (self.AllowButtons and distance < 82) then
 			local button = trace.Entity
-			if IsValid(button) and (ply:KeyDown( IN_ATTACK ) and not self.MouseDown) and button.Use then
+			if IsValid(button) and (ply:KeyDown( IN_ATTACK ) and not self.MouseDown) and !button:IsVehicle() and button.Use then
```

Comment:
Could you actually change `!` to `not`?

Reply:
Yep, all done. Sorry i've had my head buried in C# for the last few years lol!

Reply:
Hah thanks!